exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/ApiController.java

```diff
@@ -295,4 +402,17 @@ private void failureHandler(RoutingContext rc) {
     // This throwable must correspond to an internal server error.
     return Optional.empty();
   }
+
+  @VisibleForTesting
+  static Map<HashCode, String> hexEncodeTransactionMessages(
+      Map<HashCode, TransactionMessage> txMessages) {
+    return txMessages.entrySet()
```

Comment:
I hoped there is a more consice version of this operation in the standard library, but SO suggests there isn't. 
However, there is a method `Maps.transformValues`, but it returns a _lazy view_ of the given map, with the function applied whenever a value is accessed: https://google.github.io/guava/releases/27.0.1-jre/api/docs/com/google/common/collect/Maps.html#transformValues-java.util.Map-com.google.common.base.Function-